R/indentation_linter.R

```diff
@@ -241,7 +244,12 @@ indentation_linter <- function(indent = 2L, hanging_indent_style = c("tidy", "al
         type = "style",
         message = lint_messages,
         line = unname(source_expression$file_lines[bad_lines]),
-        ranges = apply(lint_ranges, 1L, list, simplify = FALSE)
+        ranges = lapply(
+          seq_along(bad_lines),
+          function(i) {
+            list(lint_ranges[i, ])
+          }
+        )
```

Comment:
`apply(..., simplify = FALSE)` is R >= 4.1, so we need to use `lapply()` instead.

Reply:
leave a comment for several years later when we can depend on R>=4.1?